consensus/types/src/chain_spec.rs

```diff
@@ -346,9 +337,10 @@ impl ChainSpec {
     }
 
     pub fn inactivity_penalty_quotient_for_fork(&self, fork_name: ForkName) -> u64 {
-        if fork_name >= ForkName::Bellatrix {
+        // TODO(superstruct_features) Is this a better pattern?
+        if fork_name.is_feature_enabled(FeatureName::Bellatrix) {
```

Comment:
Wondering if anyone has thoughts about whether this pattern is "nicer" than using `>=`?

Reply:
I think it's nicer, because `>=` wouldn't really make sense between `ForkName` and `FeatureName`